src/app/public/modules/input-box/input-box.component.ts

```diff
@@ -45,7 +45,7 @@ import {
 export class SkyInputBoxComponent implements OnInit {
 
   /**
-   * Indicates whether to visually highlight the input box in an error state. The input box also displays
+   * Indicates whether to visually highlight the input box in an error state. If not provided, the input box displays
```

Comment:
I'm not really sure that `If not provided` is meaningful for a Boolean property. I'm making a couple suggested tweaks to the docs to try to make this more clear at a glance...
```suggestion
   * Indicates whether to visually highlight the input box in an error state. If `hasErrors` is `false`, the input box still
```